src/attribution/onINP.ts

```diff
@@ -37,8 +37,12 @@ const attributeINP = (metric: INPMetric): void => {
       );
     })[0];
 
+    // TODO: remove when 1367329 is resolved
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=1367329
```

Comment:
FWIW this should be easy to test based on the repro in the crbug (`eventEntry.target` is null but `eventTarget` is non-null), which would then start failing when it was fixed in Chrome and this workaround would be ready to be removed.

Reply:
Are you offering to add a test case to this PR? Am I reading this right? 😁

However, even if fixed, we'd likely want to keep this around for a bit until that version of Chrome becomes widely used so not sure the test passing would be a good sign to remove this workaround.

Reply:
@rviscomi were you able to reliably reproduce a null target in your testing? If so maybe we can just add a test case that would fail with the current logic and then pass with the new logic (and also start failing again if we remove the fix too early).

Reply:
I think @tunetheweb has a repro here: https://www.tunetheweb.com/experiments/button/

Reply:
That repo doesn't necessarily always report the event with the missing target as the INP event. But it does show all the other events in the interaction. So good to repo the issue of missing targers, but not necessarily the issue or missing targets in INP events.

Reply:
I can take a stab. Seems straightforward

Reply:
Thanks @brendankenny!

@philipwalton @tunetheweb can we merge this PR as-is or do you want to couple it with the test case?